### ISSUE TITLE
Argument of type '{ type: "module"; deno: boolean; }' is not assignable to parameter of type 'WorkerOptions'.

### ISSUE THREAD

**Issue by srikanth-chitturi** [JamesBroadberry/deno-bcrypt#24]
I am using deno Version 1.11.1

import * as bcrypt from "https://deno.land/x/bcrypt@v0.2.4/mod.ts";
After importing bcrypt, as above , I get these errors :- 

error: TS2345 [ERROR]: Argument of type '{ type: "module"; deno: boolean; }' is not assignable to parameter of type 'WorkerOptions'.
  Object literal may only specify known properties, and 'deno' does not exist in type 'WorkerOptions'.
    { type: "module", deno: true },
                      ~~~~~~~~~~
    at https://deno.land/x/bcrypt@v0.2.4/src/main.ts:18:23

TS2345 [ERROR]: Argument of type '{ type: "module"; deno: boolean; }' is not assignable to parameter of type 'WorkerOptions'.
  Object literal may only specify known properties, and 'deno' does not exist in type 'WorkerOptions'.
    { type: "module", deno: true },
                      ~~~~~~~~~~
    at https://deno.land/x/bcrypt@v0.2.4/src/main.ts:50:23

TS2345 [ERROR]: Argument of type '{ type: "module"; deno: boolean; }' is not assignable to parameter of type 'WorkerOptions'.
  Object literal may only specify known properties, and 'deno' does not exist in type 'WorkerOptions'.
    { type: "module", deno: true },
                      ~~~~~~~~~~
    at https://deno.land/x/bcrypt@v0.2.4/src/main.ts:83:23

Found 3 errors.
[E] [daem] app crashed - waiting for file changes before starting ...

**Comment by JamesBroadberry** [comment]
Duplicate of #18

If you'd rather not use the unstable flag, please update Deno and this library (details in the README)
